deliverables/ruby_monk_problems/random_select.rb

```diff
@@ -1,3 +1,5 @@
 def random_select(array, n)
-  # your code here
+  outarray = []
+  n.times { outarray << array.shuffle[0] }
```

Comment:
In order to avoid hard-coding indices, you can reference this same concept and replace `array.shuffle[0]` with `array.shuffle.first`.